source/SymWorld.h

```diff
@@ -34,7 +34,7 @@ class SymWorld : public emp::World<Organism>{
   emp::Ptr<emp::DataMonitor<double>> data_node_hosted_syms;
   emp::Ptr<emp::DataMonitor<double>> data_node_free_syms;
   emp::Ptr<emp::DataMonitor<int>> data_node_cfu;
-
+  emp::Ptr<emp::DataMonitor<double>> data_node_sgg;
```

Comment:
Might be worth adding `emp::data::Histogram` to the template for this DataNode so that it tracks the full distribution of donation values (like the DataNodes for interactions between hosts and symbionts)